#597453: [sitecore-jss-react] fixed rendering with two and more dynamic placeholders

matkovskyi opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

- [x] This PR follows the [Contribution Guide](https://github.com/Sitecore/jss/blob/dev/CONTRIBUTING.md)
- [x] Changelog updated

## Description / Motivation
<!--- Describe your changes in detail -->
There was issue with dynamic placeholders when there uses two and more dynamic placeholders in one component.
Added conditional for correct working dynamic placeholders.
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## Testing Details
<!--- Please describe how you tested your changes. -->
<!--- When applicable, include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
- [ ] Unit Test Added
- [x] Manual Test/Other (Please elaborate)

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

illiakovalenko reviewed (changes requested):
1. Should it be a "Bug Fix" in a CHANGELOG?
2. Does the unit test cover the problem that you described?

matkovskyi commented:
> 1. Should it be a "Bug Fix" in a CHANGELOG?
> 2. Does the unit test cover the problem that you described?

1. Fixed
2. Yes, I fixed unit test because it was incorrect

illiakovalenko reviewed (approved):
👍
